examples/hello-world/hello-pt/src/hello-pt_cifar10_fl.py

```diff
@@ -43,53 +43,55 @@ def main():
         ]
     )
 
-    flare.init()
-    sys_info = flare.system_info()
-    client_name = sys_info["site_name"]
+    with FlareClientContext() as flare:
```

Comment:
It's better to create a context after an action here, instead of constructing context directly, for example,
  `with flare.init() as flare_ctx:
`
This is similar to Python's 
  `with open("test") as f:
`
This way, we can also maintain the backward compatibility.

Reply:
change as suggested.